code/modules/games/cards/playing_cards.dm

```diff
@@ -288,7 +318,26 @@
 			user.put_in_hands(H)
 		else
 			user << "<span class = 'notice'>You can't mix cards from other decks.</span>"
+	else if(istype(I, /obj/item/toy/cardhand)) //PROBLEM HERE
```

Comment:
Oh gee, that actually arose during testing and I fixed that, but never removed the comment. My mistake.